emqx/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Emqx integration with openmetrics v2",
+    "title": "EMQX",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
```

Comment:
Please also add classifier tags for categories and submitted data types, you can find the [list here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags)

Reply:
Please add one of the submitted data types from the list above